### PR TITLE
EFRS-892 Add service type on FE

### PR DESCRIPTION
**Pull request by endypanda**

### CODE REVIEW

**Review by isobolev — COMMENTED** [review]
please install prettier in your IDE, and format all touched files. You can run `npm run pretty` to format all files

File: ui/src/app/features/mode-create-dialog/model-create-dialog.component.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/*
```
Comment (isobolev) [inline ui/src/app/features/mode-create-dialog/model-create-dialog.component.ts:1]: please remove empty styles file above

File: ui/src/app/features/mode-create-dialog/model-create-dialog.component.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+@Component({
+  selector: 'app-model-create-dialog',
+  templateUrl: './model-create-dialog.component.html',
+  styleUrls: ['../create-dialog/create-dialog.component.scss','./model-create-dialog.component.scss'],
+})
+export class ModelCreateDialogComponent extends CreateDialogComponent {
+  constructor(public dialogRef: MatDialogRef<CreateDialogComponent>, @Inject(MAT_DIALOG_DATA) public data: any) {
+    super(dialogRef, data);
+  }
+  typeValues = ['RECOGNITION' , 'DETECTION'];
```
Comment (isobolev) [inline ui/src/app/features/mode-create-dialog/model-create-dialog.component.ts:29]: should be enums

Comment (endypanda) [reply ui/src/app/features/mode-create-dialog/model-create-dialog.component.ts]: changed, please check it

File: ui/src/app/features/mode-create-dialog/model-create-dialog.component.ts (hunk 1 above)
Comment (endypanda) [reply ui/src/app/features/mode-create-dialog/model-create-dialog.component.ts:1]: done

File: ui/src/app/features/mode-create-dialog/model-create-dialog.component.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+@Component({
+  selector: 'app-model-create-dialog',
+  templateUrl: './model-create-dialog.component.html',
+  styleUrls: ['../create-dialog/create-dialog.component.scss', './model-create-dialog.component.scss'],
+})
+export class ModelCreateDialogComponent extends CreateDialogComponent {
+  constructor(public dialogRef: MatDialogRef<CreateDialogComponent>, @Inject(MAT_DIALOG_DATA) public data: any) {
+    super(dialogRef, data);
+  }
+  keys = Object.keys;
```
Comment (isobolev) [inline ui/src/app/features/mode-create-dialog/model-create-dialog.component.ts:29]: please use keyvalue pipe instead in template

**Review by isobolev — APPROVED** [review]

**Comment by endypanda** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**